packages/js-dpp/lib/errors/consensus/basic/dataContract/IncompatibleDataContractSchemaError.js

```diff
@@ -0,0 +1,107 @@
+const AbstractBasicError = require('../AbstractBasicError');
+const Identifier = require('../../../../identifier/Identifier');
+
+/**
+ * Parse schema validation error message
+ *
+ * @param {Error} schemaValidationError
+ *
+ * @returns {{ op: string, path: string }[]}
+ */
+function parseSchemaValidationError(schemaValidationError) {
+  const regexp = /change = (.*?)$/;
+
+  const match = schemaValidationError.message.match(regexp);
+
+  return JSON.parse(match[1]);
+}
+
+class IncompatibleDataContractSchemaError extends AbstractBasicError {
+  /**
+   * @param {Buffer} dataContractId
+   * @param {Error} schemaValidationError
```

Comment:
Not sure cbor will encode and decode error class properly. Maybe we need to pass operation and path such as in immutability error?